Formula/zig.rb

```diff
@@ -0,0 +1,30 @@
+class Zig < Formula
+  desc "Programming language designed for robustness, optimality, and clarity"
+  homepage "http://ziglang.org/"
+  url "https://s3.amazonaws.com/ziglang.org/download/0.1.1/zig-0.1.1.tar.xz"
+  sha256 "a160d14aecebead2f8c574b0dd687cdfecc53d149d521a17b89b629d8b558c94"
+
+  depends_on "cmake" => :build
+  depends_on "llvm@5" => :build
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    (testpath/"hello.zig").write <<~HEREDOC
```

Comment:
We'll prefer `EOS` to `HEREDOC`, for consistency with other formulas.
And can the empty lines in `hello.zig` be removed?

Reply:
Sure

Reply:
til https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc